packages/verkle/src/node/internalNode.ts

```diff
@@ -47,35 +53,47 @@ export class InternalNode extends BaseVerkleNode<VerkleNodeType.Internal> {
     // TODO: Generate Point from rawNode value
     const commitment = rawNode[rawNode.length - 1]
 
-    return new InternalNode({ commitment, depth })
+    return new InternalNode({ commitment, depth, verkleCrypto })
   }
 
-  static create(depth: number): InternalNode {
+  static create(depth: number, verkleCrypto: VerkleCrypto): InternalNode {
     const node = new InternalNode({
       commitment: POINT_IDENTITY,
       depth,
+      verkleCrypto,
     })
 
     return node
   }
 
-  getChildren(index: number): VerkleNode | null {
+  getChildren(index: number): Uint8Array | null {
     return this.children?.[index] ?? null
   }
 
-  insert(key: Uint8Array, value: Uint8Array, resolver: () => void): void {
+  insert(
+    key: Uint8Array,
+    value: Uint8Array,
+    resolver: () => void,
+    verkleCrypto?: VerkleCrypto
+  ): void {
     const values = new Array<Uint8Array>(NODE_WIDTH)
     values[key[31]] = value
-    this.insertStem(key.slice(0, 31), values, resolver)
+    this.insertStem(key.slice(0, 31), values, resolver, verkleCrypto!)
   }
 
-  insertStem(stem: Uint8Array, values: Uint8Array[], resolver: () => void): void {
+  insertStem(
+    stem: Uint8Array,
+    values: Uint8Array[],
+    resolver: () => void,
+    verkleCrypto: VerkleCrypto
+  ): void {
     // Index of the child pointed by the next byte in the key
     const childIndex = stem[this.depth]
 
     const child = this.children[childIndex]
 
     if (child instanceof LeafNode) {
+      // TODO: Understand the intent of what cowChild is suppoded to do
```

Comment:
It seems like we have no way of "stacking commitments" (perhap because of what we have access to through the verkle cryptography wasm API), so I guess we can do without `cowChild` for the time being? We can always re-add this at some point if we have the need for it.

Reply:
I would tend to agree.  We probably will be able to improve our performance down the line but would prefer to do everything naively for starters.

Reply:
agreed!